strings-csharp/HexToByteArray/Tests/Test.cs

```diff
@@ -0,0 +1,88 @@
+using HexToByteArray;
+
+namespace Tests
+{
+    public class Test
+    {
+        private readonly ConvertHex _convertHex;
+        private const string HexString = "FF008000";
+        public Test()
+        {
+            _convertHex = new ConvertHex();
+        }
+
+        [Fact]
+        public void WhenValidHexPassed_ThenConverionSuccessful()
+        {
+            byte[] expectedResult = { 255, 0, 128, 0 };
+            var result = _convertHex.FromHexString(HexString);
+
+            Assert.Equal(expectedResult, result);
+        }
+
+        [Fact]
+        public void WhenInvalidFormatHexPassed_ThenConverionFail()
+        {
+            var expectedResult = "FormatException";
+            try
+            {
+                var result = _convertHex.FromHexString("Hello");
+            }
+            catch (Exception ex)
+            {
+                Assert.Equal(expectedResult, ex.GetType().Name);
```

Comment:
xUnit has exception assertions, you should use them. The syntax is:
Assert.Throws<exT>, so in this case it would be:
Assert.Throws<FormatException>(() => _convertHex.FromHexString("Hello"));

Reply:
Done. Thanks!